### ISSUE TITLE
3.9 version upcoming?

### ISSUE THREAD

**Issue by pjonker** [moodleuulm/moodle-local_boostnavigation#63]
Hello guys, since Moodle 3.9 is coming up, do you have any plans of upgrading the plugin to a 3.9 version?

**Comment by abias** [comment]
Hi @pjonker ,

please see our release support information on https://github.com/moodleuulm/moodle-local_boostnavigation/blob/master/README.md#moodle-release-support.

To summarize: Yes, we will update and test this plugin for 3.9. But this will take some time, especially during demanding times like we have currently.

You can help us by testing the latest release on 3.9 and by reporting any problems which you encounter.

Thanks for your patience,
Alex

**Comment by pjonker** [comment]
Dear Alex,

I tested the Local Boost Navigation Fumbling for Pjonker in Moodle 3.9.
I've compared it with a 3.7 version. The only thing I've discovered that
doesn't seem to work in 3.9 is the collapsable node. Even when you turn
this on it remains uncollapsed. All the other options do seem to work.


With kind regards,
Jaron Cohen

*Consultant*
j.cohen@avetica.nl| 06 - 41 23 57 09
afwezig op woensdag


*Avetica B.V.*
Sportlaan 3D
3299 XG Maasdam
KvK: 52831094
+31 (0)88 - 66 44 800
www.avetica.nl



ISO 9001 | ISO 27001 | ISAE 3000
*Gecertificeerd Premium Moodle Partner*

Op ma 29 jun. 2020 om 16:25 schreef Peter Jonker <p.jonker@avetica.nl>:
